docs/source/sequences.rst

```diff
@@ -42,4 +42,35 @@ Lightning can handle TBTT automatically via this flag.
     override :meth:`pytorch_lightning.core.LightningModule.tbptt_split_batch`.
 
 .. note:: Using this feature requires updating your LightningModule's :meth:`pytorch_lightning.core.LightningModule.training_step` to include
-    a `hiddens` arg.
\ No newline at end of file
+    a `hiddens` arg.
+
+Iterable Datasets
+---------------------------------------
+Lightning supports using IterableDatasets as well as map-style Datasets. IterableDatasets provide a more natural
+option when using sequential data.
+
+.. note:: When using an IterableDataset you must set the val_check_interval when initializing the Trainer even when
```

Comment:
Can we say more here - specifically that `val_check_interval` must be an `int` and not a `float`?

Reply:
Absolutely. Do you think there should be more in the description or just specify the type?

Reply:
Perhaps something like 'must set the val_check_interval to be an int (specifying the number of training batches to run before performing validation)' or words to that effect :)